test/e2e/test_system.py

```diff
@@ -373,6 +374,17 @@ def check_patch_system_response_success(self, expected_system_get_data: dict):
         assert self._patch_response.status_code == 200
         assert self._patch_response.json() == expected_system_get_data
 
+        # pylint:disable=fixme
+        # TODO: Move the below code into a utility or something later - will be wanted for other tests?
+        original_data = self._post_response.json()
+        new_data = self._patch_response.json()
+
+        # Created time should be unchanged, but new modified time should be greater than before
+        assert original_data["created_time"] == new_data["created_time"]
+        assert datetime.fromisoformat(new_data["modified_time"]) > datetime.fromisoformat(
+            original_data["modified_time"]
+        )
```

Comment:
Perhaps an e2e test helpers in conftest?

Reply:
Yeah I guess so if it can be reused later.